htdocs/product/price.php

```diff
@@ -283,6 +283,9 @@
 		$psq = empty($newpsq) ? 0 : $newpsq;
 		$maxpricesupplier = $object->min_recommended_price();
 
+		// Packaging / Conditionnement
```

Comment:
When adding comments in french in the code, I suggest to at least indicate they are in french:

```php
// Packaging (EN) / Conditionnement (FR)
```